numpy/core/fromnumeric.py

```diff
@@ -1004,7 +1004,9 @@ def _argsort_dispatcher(a, axis=None, kind=None, order=None):
 @array_function_dispatch(_argsort_dispatcher)
 def argsort(a, axis=-1, kind=None, order=None):
     """
-    Returns the indices that would sort an array.
+    Returns the indices that would sort an array. 
+    The returned indices can then be used as a_sorted = a[returned indices] which will now give us the sorted array 'a'.
+    The zeroth number in [returned indices] is a the index of the smallest number in array 'a'.    
```

Comment:
I think this should read 

```suggestion
    The zeroth number in [returned indices] is the index of the smallest number in array 'a'.    
```